db/migrate/20220525005938_add_urls_to_organization.rb

```diff
@@ -0,0 +1,7 @@
+class AddUrlsToOrganization < ActiveRecord::Migration[6.1]
+  def change
+    t.string :facebook_url
+    t.string :twitter_url
+    t.string :instagram_url
```

Comment:
En principio usaría un nombre para descriptivo para la migración como `20220525005938_add_social_media_urls_to_organizations.rb`
Por otra parte, el método para agregar columnas a una tabla es `add_column`
```suggestion
add_column :organizations, :facebook_url, :string
add_column :organizations, :twitter_url, :string
add_column :organizations, :instagram_url, :string
```